s3torchconnector/tst/e2e/models/net.py

```diff
@@ -0,0 +1,31 @@
+import torch
```

Comment:
Missing copyright. Though are these files something Amazon owns, or did you copy/paste this from an example?

Reply:
This was previously added as part of the checkpoint work, but I think it's from an example too - https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html, that's why I removed the copyright, but missed removing it in my class.